custom-elements/HTMLElement-constructor.html

```diff
@@ -28,6 +28,18 @@
 }, 'HTMLElement constructor must throw TypeError when it has not been defined by customElements.define');
 
 test(function () {
+    class SomeCustomElement extends HTMLParagraphElement {};
+    customElements.define('some-custom-element', SomeCustomElement);
+    assert_throws({'name': 'TypeError'}, function () { new SomeCustomElement(); });
+}, 'Custom element constructor must throw TypeError when it does not extend HTMLElement');
+
+test(function () {
+    class SomeCustomButtonElement extends HTMLButtonElement {};
+    customElements.define('some-custom-button-element', SomeCustomButtonElement, { extends: "p" });
```

Comment:
It would be nice to be consistent about using single or double quotes for string literals.